express-post-json/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const arr = [];
```

Comment:
This array should only exist in the get request. It should be created new each time and sent to the client with the current grades objects